c++/src/kj/compat/http.c++

```diff
@@ -3373,8 +3396,11 @@ private:
           KJ_IF_SOME(m, maxSize) {
             // This is only non-null for `receive` calls, so we must be decompressing. We don't want
             // the decompressed message to OOM us, so let's make sure it's not too big.
-            KJ_REQUIRE(totalBytesProcessed < m,
-                "Decompressed WebSocket message is too large");
+            if (totalBytesProcessed > m) {
+              return ProtocolError {
```

Comment:
Can you use `WebSocket::ProtocolError`, instead of `ProtocolError` which is meant for HTTP errors?

Reply:
Yea sure, it's using WebSocket::ProtocolError because the zlibcontext is defined inside of the websocket class but I can change it so that it's clearer we're using WebSocket's one

Reply:
Oh! In that case it's fine. I misread.